x-pack/plugins/reporting/server/lib/store/report.ts

```diff
@@ -58,6 +60,8 @@ export class Report implements Partial<ReportSource> {
     this._primary_term = opts._primary_term;
     this._seq_no = opts._seq_no;
```

Comment:
Based on this question, I added this commit: 71e89ea587d 

The `_` mimics the field that ES uses to track concurrency to updates on the document. It does not denote public/private status of the class field. They are interspersed with fields that do not have an underscore because this is a flattened version what ES returns when you `GET` a document. 

https://www.elastic.co/guide/en/elasticsearch/reference/7.13/optimistic-concurrency-control.html
> Elasticsearch keeps tracks of the sequence number and primary term of the last operation to have changed each of the documents it stores. The sequence number and primary term are returned in the _seq_no and _primary_term fields in the response of the GET API